rskj-core/src/main/java/co/rsk/signing/Message.java

```diff
@@ -0,0 +1,17 @@
+package co.rsk.signing;
+
+import org.spongycastle.util.encoders.Hex;
+
+/**
+ * Represents a message that can be signed.
+ *
+ * @author Ariel Mendelzon
+ */
+public abstract class Message {
```

Comment:
This can also be an `interface` with a `default` method... Not saying that it's better, but you can take a look!

Reply:
Not taken, leaving as is.